Lambda.dfy(24,11): Error: assertion could not be proved

Dafny program verifier finished with 3 verified, 1 error
